tests/reader.test.js

```diff
@@ -0,0 +1,224 @@
+const { expect } = require("chai");
+const request = require("supertest");
+const { Reader } = require("../src/models");
+const app = require("../src/app");
+const faker = require("faker");
+
+describe("/readers", () => {
+  before(async () => Reader.sequelize.sync());
+
+  beforeEach(async () => {
+    await Reader.destroy({ where: {} });
+  });
+
+  describe("with no records in the database", () => {
+    describe("POST /readers", () => {
+      it("creates a new reader in the database", async () => {
+        const response = await request(app).post("/readers").send({
+          name: "Elizabeth Bennet",
+          email: "future_ms_darcy@gmail.com",
+          password: "123456789",
+        });
+        const newReaderRecord = await Reader.findByPk(response.body.id, {
+          raw: true,
+        });
+
+        expect(response.status).to.equal(201);
+        expect(response.body.name).to.equal("Elizabeth Bennet");
+        expect(newReaderRecord.name).to.equal("Elizabeth Bennet");
+        expect(newReaderRecord.email).to.equal("future_ms_darcy@gmail.com");
+      });
+
+      it("Should star out password when creating a new reader", async () => {
+        const reader = {
+          name: faker.name.firstName(),
+          email: faker.internet.email(),
+          password: "123456789",
+        };
+        const response = await request(app).post("/readers").send(reader);
+        expect(response.status).to.equal(201);
+        expect(response.body.password).to.equal("*********");
+      });
+
+      it("Should return a 500 if name is empty", async () => {
+        const readerData = {
+          name: "",
+          email: faker.internet.email(),
+          password: "12345567890abc",
+        };
+
+        const result = await request(app).post("/readers").send(readerData);
+        expect(result.status).to.equal(500);
+        expect(result.body.error).to.equal(
+          "Validation error: Name can not be empty"
+        );
+      });
```

Comment:
Leave a clear line between each `it` block
```suggestion
      });

```